public/js/createProduct.js

```diff
@@ -0,0 +1,32 @@
+// Handle creating a new project
+const createProductHandler = async (event) => {
+    event.preventDefault();
+
+    // Dummy project data for testing/development
+    let product = {
+        "product_name": "Some Product",
+        "description": "Some Description",
+        "img_url": "url",
+        "location_zipcode": "27510",
+        "category_id": "1",
+        "user_id": "2"
```

Comment:
These values will eventually come from frontend input fields, similar to `./login ` or `./updateInfo`.